bucket/git-filter-repo.json

```
@@ -7,12 +7,18 @@
         "Git": "git",
         "Python 3": "python"
     },
-    "url": "https://github.com/newren/git-filter-repo/archive/v2.47.0.zip#/dl.7z",
+    "url": "https://github.com/newren/git-filter-repo/archive/v2.47.0.zip#/dl.bin",
```

```suggestion
    "url": "https://github.com/newren/git-filter-repo/archive/v2.47.0.zip#/dl.zi_",
```

Should use a meaningful extension